legacy/c3_trajectory_generator/src/waypoint_validity.cpp

```diff
@@ -0,0 +1,61 @@
+#include "waypoint_validity.hpp"
+
+// Point and sub_size must be relative to ogrid, IE: meters * 1/OGRID_RESOLUTION + offset
```

Comment:
Then it would be clearer to say that the sub_size is in units of map cells. (meters cancel out)